packages/main/src/StepInput.hbs

```diff
@@ -0,0 +1,68 @@
+<div
+	id="{{_id}}"
+	class="ui5-step-input-root"
+	style="{{styles}}"
+	@keydown="{{_onkeydown}}"
+	@focusin="{{_onfocusin}}"
+	@focusout="{{_onfocusout}}"
+>
+	<div
+		class="ui5-step-icon ui5-step-dec"
+		title="{{decIconTitle}}"
+		?interactive="{{_decIconInteractive}}"

```

Comment:
`interactive` is a property of the ui5-icon, here it is set to a "div" element, here and below for the second icon wrapper.

Reply:
Removed. It was leftover from some tests.